lib/data/managers/firestore_manager.dart

```diff
@@ -45,8 +46,10 @@ class FireStoreManager {
               .document(stagesPathReference)
               .get()
               .then((stagesSnapshot) async {
-            if (stagesSnapshot.data != null && stagesSnapshot.data["status"] == DataStatus.PUBLISHED) {
-              crop.addStage(Stage.stageFromDocument(stagesSnapshot));
+            if (stagesSnapshot.data != null &&
+                stagesSnapshot.data[documentFieldStatus] ==
+                    DataStatus.PUBLISHED) {
```

Comment:
Can we move `DataStatus.PUBLISHED` to the same line as `stagesSnapshot.data[documentFieldStatus] ==`
So, we would have:
`stagesSnapshot.data[documentFieldStatus] == DataStatus.PUBLISHED`

Reply:
It has happened because of IDE's auto-indentation but now is solved.